cmake/FindHPX_BLAS.cmake

```diff
@@ -4,9 +4,16 @@
 # Distributed under the Boost Software License, Version 1.0. (See accompanying
 # file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
 
-hpx_find_package(BLAS
-  LIBRARIES blas libblas
-  LIBRARY_PATHS lib64 lib
-  HEADERS cblas.h
-  HEADER_PATHS include)
-
+hpx_find_package(
+  BLAS
+  LIBRARIES
+  blas
+  libblas
+  LIBRARY_PATHS
+  lib64
+  lib
+  HEADERS
+  cblas.h
+  HEADER_PATHS
+  include
```

Comment:
Questionable if this is better...

Reply:
Yah, I might have missed adding `hpx_find_package` to the list of custom commands...

Reply:
Actually, `hpx_find_package` does not exist anymore and those files are all obsolete. I'll remove them.